content/en/docs/release-notes/release-notes-0.15.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Release Notes"
+linkTitle: "v0.15"
+weight: 860
+type: "docs"
+---
+
+The `v0.15` release has a few focus areas:
+
+* Experimental improved controllers
+* Support for OpenShift's OLM
+* Improved deployment of the webhook
+* General Availability of JKS and PKCS#12 keystores
+* cmctl CLI tool
```

Comment:
Need to remove references to the name `cmctl`

Reply:
Renamed